uiServices/src/homepage_test.go

```diff
@@ -1,61 +1,53 @@
 package services
 
-import (
-	"github.com/go-chi/chi"
-	"github.com/stretchr/testify/assert"
-	"net/http"
-	"net/http/httptest"
-	"testing"
-)
+// func TestStartUiModeSuccesfully(t *testing.T) {
```

Comment:
@jenny-flynn These tests rely on the ui distribution folder being generated along with index.html. I don't think they are adding much value maybe , could we remove them. Unless you have ideas on how to save them .

Reply:
I was also the one that asked @CameliaFarcas to comment them out so that we could get your opinion

Reply:
yep, i've no issues with this being removed. it's tested in the godog tests anyway

Reply:
@CameliaFarcas can you remove these?